reference/api/index.rst

```diff
@@ -0,0 +1,180 @@
+Web API
+=======
+
+.. toctree::
+
+    rest
+    jobs
+
+How it's structured
+-------------------
+
+There is actually two APIs. A REST API with basic CRUD operations, and a jobs API which can launch heavy processes asynchronously.
+The current version of the API is v1. All URI have to request explicitly this version.
+
+.. code-block:: bash
+
+    GET https://demo.akeneo.com/api/rest/v1
+    GET https://demo.akeneo.com/api/jobs/v1
+
+Request these URIs to get all the endpoint categories that the API supports.
+
+Scope of the API
+----------------
+
+Currently, only Product, Category, Family, Attribute and Attribute Option entities are covered by the API.
+
+.. warning::
+    As these APIs are mainly designed to be used by connectors, Enterprise Edition permissions are not managed.
+
+Security
+--------
+
+We use `OAuth2 <https://oauth.net/2/>`_ to authenticate users in the API and
+`Symfony ACLs <http://symfony.com/doc/2.7/security/acl.html>`_ to handle authorizations.
+
+Authentication
+~~~~~~~~~~~~~~
+
+Creating an OAuth client
+************************
+
+The first operation you will need to authorize third-party applications to use the API is to create a `client id`.
+To do so, use the following command:
+
+.. code-block:: bash
+
+    php bin/console pim:oauth-server:create-client \
+        --grant-type="password" \
+        --grant-type="refresh_token"
+
+You will receive the client id and client secret:
+
+.. code-block:: bash
+
+    A new client with client id 4gm4rnoizp8gskgkk080ssoo80040g44ksowwgw844k44sc00s and secret 5dyvo1z6y34so4ogkgksw88ookoows00cgoc488kcs8wk4c40s has been added.
+
+These keys must be transmitted by the administrator to any third-party application wanting to use the API.
+
+Client ids are not enough to access the API, it's the role of tokens. Client ids allow two things:
```

Comment:
You're right, the missing info is that a client id always comes with a secret. Maybe we just need to add a sentence about that instead of modifying all the others?